symphony-bdk-core/src/test/java/com/symphony/bdk/core/OboServicesTest.java

```diff
@@ -0,0 +1,45 @@
+package com.symphony.bdk.core;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.mock;
+
+import com.symphony.bdk.core.auth.AuthSession;
+import com.symphony.bdk.core.config.model.BdkConfig;
+
+import com.symphony.bdk.core.service.message.MessageService;
+import com.symphony.bdk.core.service.stream.StreamService;
+
+import com.symphony.bdk.core.service.user.UserService;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class OboServicesTest {
+
+  private OboServices oboServices;
+
+  @BeforeEach
+  void setUp() {
+    oboServices = new OboServices(new BdkConfig(), mock(AuthSession.class));
+  }
+
+  @Test
+  void testOboStreams() {
+    assertNotNull(oboServices.streams());
+    assertEquals(StreamService.class, oboServices.streams().getClass());
```

Comment:
How can we assert the `OboStreamService.class` interface is returned by method `oboServices.streams()` and not directly `StreamService.class`?